install/k8s/contiv/contiv.yaml

```diff
@@ -1,4 +1,99 @@
 ---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: contiv-netplugin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: contiv-netplugin
+subjects:
+- kind: ServiceAccount
+  name: contiv-netplugin
+  namespace: kube-system
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: contiv-netplugin
+  namespace: kube-system
+rules:
+  - apiGroups:
+    - ""
+    - extensions
+    resources:
+      - endpoints
+      - nodes
+      - namespaces
+      - networkpolicies
+      - pods
+      - services
+    verbs:
+      - watch
+      - list
+      - update
+      - get
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Same question here too. Contiv works with the default `ServiceAccount` cert in (`/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`). Why is this needed?

https://github.com/contiv/netplugin/blob/master/Makefile#L18

Reply:
That's CA, not token, I think token is the one indicates k8s role and accounts

Reply:
Every `ServiceAccount` translates to a `ca.crt` file on disk:

https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/